region/lowernorfair/east.json

```diff
@@ -1491,7 +1498,52 @@
           "to": [
             {
               "id": 1,
-              "requires": [],
+              "requires": [
+                {"or": [
+                  "heatProof",
+                  "canHeatRun"
+                ]},
+                {"or": [
+                  {"and": [
+                    "Charge",
+                    "Plasma",
+                    {"or": [
+                      {"cost": [
+                        {"heatFrames": 1200}
+                      ]},
+                      {"and": [
+                        "Ice",
+                        {"or": [
+                          {"cost": [
+                            {"heatFrames": 950}
+                          ]},
+                          {"and": [
+                            "Wave",
+                            {"cost": [
+                              {"heatFrames": 700}
+                            ]}
+                          ]}
+                        ]}
+                      ]}
+                    ]}
+                  ]},
+                  {"and": [
+                    "Charge",
+                    "Spazer",
+                    "Ice",
+                    "Wave",
+                    {"cost": [
+                      {"heatFrames": 2000}
+                    ]}
+                  ]},
+                  {"and": [
+                    "Super",
+                    {"cost": [
+                      {"heatFrames": 700}
+                    ]}
+                  ]}
```

Comment:
Should we add the speed echoes strat in there too? I feel like it should be locked behind a tech of some kind but it is a quick way to kill them.

Reply:
I couldn't get it done. Overall it seems pretty useless as it's quicker with 6 supers but in the rare case one don't have supers it could be a thing. Probably gonna need a video though.